doc/source/howtos/virt_sandbox.rst

```diff
@@ -141,169 +102,259 @@ with application servers.
 
-      Clustercheck will now listen on port 9200 after ``xinetd`` restarts
-      and HAProxy is ready to check MySQL via HTTP::
+     mysql> SELECT * FROM monitor.mysql_server_connect_log ORDER BY time_start_us DESC LIMIT 6;
+     +---------------+------+------------------+----------------------+---------------+
+     | hostname      | port | time_start_us    | connect_success_time | connect_error |
+     +---------------+------+------------------+----------------------+---------------+
+     | 192.168.70.61 | 3306 | 1469635762434625 | 1695                 | NULL          |
+     | 192.168.70.62 | 3306 | 1469635762434625 | 1779                 | NULL          |
+     | 192.168.70.63 | 3306 | 1469635762434625 | 1627                 | NULL          |
+     | 192.168.70.61 | 3306 | 1469635642434517 | 1557                 | NULL          |
+     | 192.168.70.62 | 3306 | 1469635642434517 | 2737                 | NULL          |
+     | 192.168.70.63 | 3306 | 1469635642434517 | 1447                 | NULL          |
+     +---------------+------+------------------+----------------------+---------------+
+     6 rows in set (0.00 sec)
 
-         # service xinetd restart
+  .. code-block:: text
 
-If you did everything correctly,
-the statistics page for HAProxy should look like this:
+     mysql> SELECT * FROM monitor.mysql_server_ping_log ORDER BY time_start_us DESC LIMIT 6;
+     +---------------+------+------------------+-------------------+------------+
+     | hostname      | port | time_start_us    | ping_success_time | ping_error |
+     +---------------+------+------------------+-------------------+------------+
+     | 192.168.70.61 | 3306 | 1469635762416190 | 948               | NULL       |
+     | 192.168.70.62 | 3306 | 1469635762416190 | 803               | NULL       |
+     | 192.168.70.63 | 3306 | 1469635762416190 | 711               | NULL       |
+     | 192.168.70.61 | 3306 | 1469635702416062 | 783               | NULL       |
+     | 192.168.70.62 | 3306 | 1469635702416062 | 631               | NULL       |
+     | 192.168.70.63 | 3306 | 1469635702416062 | 542               | NULL       |
+     +---------------+------+------------------+-------------------+------------+
+     6 rows in set (0.00 sec)
 
-.. image:: ../_static/pxc_haproxy_status_example.png
+  The previous examples show that ProxySQL is able to connect and ping the nodes
+  you added.
+
+  To enable monitoring of these nodes, load them at runtime:
+
+  .. code-block:: text
+
+     mysql> LOAD MYSQL SERVERS TO RUNTIME;
+
+  .. _proxysql-client-user:
+
+  .. rubric:: Creating ProxySQL Client User
+
+  ProxySQL must have users that can access backend nodes
+  to manage connections.
+
+  To add a user, insert credentials into ``mysql_users`` table:
+
+  .. code-block:: text
+
+     mysql> INSERT INTO mysql_users (username,password) VALUES ('sbuser','sbpass');
+     Query OK, 1 row affected (0.00 sec)
+
+  .. note::
+
+     ProxySQL currently doesn't encrypt passwords.
```

Comment:
ProxySQL now supports pain pwd and mysql native plugin hash password. proxysql2 do not support `caching_sha2_password` plugin for the application users. https://github.com/sysown/proxysql/issues/2580